### PR TITLE
Press-event, href and target

### PR DESCRIPTION
**Pull request by jfrolich**
This adds `href` and `target` for React-Native-Web (pretty important for accessibility and SEO). Additionally it fixes a bug where the press event was not passed in a `Text` component. This is related, because when you have client routing you'd want to provide a `href` and do a `preventDefault` on the press event.

### CODE REVIEW

File: src/components/ScrollView.re [hunk 1]
```diff
@@ -137,7 +137,9 @@ external make:
     ~shouldRasterizeIOS: bool=?,
     ~style: Style.t=?,
     ~testID: string=?,
-    ~children: React.element=?
+    ~children: React.element=?,
+    // temporary react-native-web-prop
+    ~useWindowScrolling: bool=?
```
Comment (MoOx) [inline src/components/ScrollView.re]: This thing doesn't seems to be in RNW repo. Can you move this back & keep it in a patch if you use a specific hacked RNW version? :D

File: src/components/Text.re [hunk 2]
```diff
@@ -55,8 +55,8 @@ external make:
     ~ellipsizeMode: [@bs.string] [ | `clip | `head | `middle | `tail]=?,
     ~numberOfLines: int=?,
     ~onLayout: Event.layoutEvent => unit=?,
-    ~onLongPress: unit => unit=?,
-    ~onPress: unit => unit=?,
+    ~onLongPress: Event.pressEvent => unit=?,
+    ~onPress: Event.pressEvent => unit=?,
```
Comment (MoOx) [inline src/components/Text.re]: 😚

File: src/components/ScrollView.re (hunk 1 above)
Comment (MoOx) [inline src/components/ScrollView.re]:
```suggestion
    ~children: React.element=?
```

Comment (MoOx) [reply src/components/ScrollView.re]: I have removed it, be sure to add it in your personal patch!

Comment (jfrolich) [reply src/components/ScrollView.re]: Sorry forget this PR was on my master branch! This was a personal patch for a PR that solves a current issue in react-native-web with window scrolling.

Normally I create a branch specifically for PR's will do again for the next PR :D

File: src/components/Text.re [hunk 3]
```diff
@@ -69,6 +69,25 @@ external make:
...
+            | `next
+            | `pingback
+            | `preconnect
+            | `prefetch
+            | `preload
+            | `prerender
+            | `prev
+            | `search
+            | `stylesheet
+          ]
```
Comment (MoOx) [inline src/components/Text.re:87]: @jfrolich curious to know where you find this list :) 
I saw some missings (eg: nofollow, noopenner & more...) and some incorrect (eg: preload, stylesheet and some others). I am referring to https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel (2nd column)

**Pull request merged**